components/training/eventBooking.tsx

```diff
@@ -43,6 +43,7 @@ export const EventBooking: FC<EventBookingType> = ({ data }) => {
             event={event}
             count={data.eventList.length}
             index={index}
+            duration={data?.duration}
```

Comment:
what is duration? is isn't clear what scale it is representing

Reply:
Its event's duration, do you want to rename it?

Reply:
again, not sure if duration means minutes, hours, days?

i'd suggest renaming it so that it is clear

Reply:
Done - I just renamed it to **eventDurationInDays**